scripts/uberenv/vcpkg_ports/axom/CONTROL

```diff
@@ -1,4 +1,9 @@
 Source: axom
 Version: develop
 Description: Host-config generator for Axom TPLs
-Build-Depends: hdf5, conduit
+Build-Depends: hdf5, conduit, mfem
+Default-Features: hdf5, conduit, mfem
+
+Feature: mfem
+Description: Adds basic (serial) mfem support to Axom
+Build-Depends: mfem
```

Comment:
I added ``mfem`` as a ``Feature`` (similar to ``spack``'s ``Variant``) so allow disabling ``mfem``.

I have not yet implemented the ability to disable ``mfem`` in the ``portfile.cmake``.  I'll take care of that in a future PR.